pkg/cmd/repo/clone/clone.go

```diff
@@ -37,8 +38,13 @@ func NewCmdClone(f *cmdutil.Factory, runF func(*CloneOptions) error) *cobra.Comm
 	cmd := &cobra.Command{
 		DisableFlagsInUseLine: true,
 
-		Use:   "clone <repository> [<directory>] [-- <gitflags>...]",
-		Args:  cobra.MinimumNArgs(1),
+		Use: "clone <repository> [<directory>] [-- <gitflags>...]",
+		Args: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				return &cmdutil.FlagError{Err: errors.New("cannot clone: no repository name given")}
```

Comment:
Tiny change 😉. The message could be changed to "repo clone: Error: Repository name is required to clone"

Reply:
@mislav Any suggestions on the copy?